### PR TITLE
docs(ja): translate guides/components-glossary/pages-fetch

### PR DESCRIPTION
**Pull request by inouetakuya**
@aytdm @53able Could you review this PR?

refs

- https://github.com/nuxt/nuxtjs.org/issues/544
- https://github.com/vuejs-jp/ja.nuxtjs.org/issues/22

### CODE REVIEW

File: content/ja/guides/components-glossary/pages-fetch.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
 
 ## Nuxt >= 2.12
 
-Nuxt.js `v2.12` introduces a new hook called `fetch` **in any of your Vue components**.
+Nuxt.js `v2.12` では、**どの Vue コンポーネントにも** `fetch` という新しいフックが導入されています。
 
 <base-alert>
 
-`fetch(context)` has been deprecated, instead you can use an [anonymous middleware](/guides/components-glossary/pages-middleware#anonymous-middleware) in your page: `middleware(context)`
+`fetch(context)` メソッドは非推奨となりましたが、代わりに [無名ミドルウェア](/guides/components-glossary/pages-middleware#無名ミドルウェア) をページコンポーネント内の `middleware(context)` で使うことできます。
```
Comment (aytdm) [inline content/ja/guides/components-glossary/pages-fetch.md:14]: 半角スペースが不要かもしれません

```suggestion
`fetch(context)` メソッドは非推奨となりましたが、代わりに[無名ミドルウェア](/guides/components-glossary/pages-middleware#無名ミドルウェア)をページコンポーネント内の `middleware(context)` で使うことできます。
```

**Review by aytdm — COMMENTED** [review]
翻訳ありがとうございます。コメントしました

**Comment by inouetakuya** [comment]
@aytdm Thank you for your suggestions, and which are committed :octocat:

**Comment by inouetakuya** [comment]
@debs-obrien This PR is ready for merging :octocat:

**Pull request merged**